src/wmtk/operations/Operation.cpp

```diff
@@ -50,10 +51,12 @@ bool Operation::after() const
 
 void Operation::update_cell_hash(const std::vector<Tuple>& cells)
 {
-    auto acc = m_mesh.get_cell_hash_accessor();
-    for (const Tuple& t : cells) {
-        ++acc.scalar_attribute(t);
-    }
+    m_mesh.update_cell_hashes(cells, m_hash_accessor);
+}
+
+Tuple Operation::resurrect_tuple(const Tuple& tuple)
```

Comment:
we need to be extra careful that teh difference betwen `update_cell_hash` and `resurrect_tuple` (or whatever other name we come up with) have very obviously distinct names to indicate their purposes

Reply:
also, why can't this be const?

Reply:
I forgot the `const`. It's there now.